### PR TITLE
update_0.30.0

### PR DESCRIPTION
**Pull request by sannykr**

### CODE REVIEW

**Comment by mariusvniekerk** [comment]
yeah this one needs to be rerendered and turned into a noarch

**Comment by sannykr** [comment]
@mariusvniekerk : Thanks. I will rerender and turn it into a noarch.

**Comment by sannykr** [comment]
Is it good to rebase the commits into 2 commits -  1) version bump .   2) rerender  ?

**Comment by jakirkham** [comment]
Needs a re-render after the noarch change.

Also the email on those commit messages is still off. Do you know how to fix it? 😕

**Comment by sannykr** [comment]
@jakirkham : Thanks for letting me know. I will fix it now

**Comment by sannykr** [comment]
I have added the email using my terminal. It should be seen in the commits that I make now.

**Comment by sannykr** [comment]
@jakirkham : I have added the email to the terminal. Could you please merge it now.

**Comment by jakirkham** [comment]
Sorry to be a pain, but do you mind correcting the author email in the last two commits as well?

**Comment by sannykr** [comment]
@jakirkham : Thanks for the feedback. I have updated the commits by making a force push.

**Comment by jakirkham** [comment]
Thanks. Also we need a re-rendering after adding noarch to update the CI configuration files.

**Comment by sannykr** [comment]
I tried to rerender, but it says there are no changes after re-rendering. I had already re-rerenderd in previous commits before noarch.

File: recipe/meta.yaml [hunk 1]
```diff
@@ -13,6 +13,7 @@ source:
 
 build:
   number: 0
+  python: noarch
```
Comment (jakirkham) [inline recipe/meta.yaml:16]: So this should be flipped. Should be `noarch: python`.

**Comment by jakirkham** [comment]
So there was an issue that cropped up above. Please address and then try re-rendering again.

**Comment by sannykr** [comment]
There are so many commits. If you want I can create a different branch with only two commits (version bump and re-render).

File: recipe/meta.yaml [hunk 2]
```diff
@@ -30,7 +31,6 @@ test:
   imports:
     - wheel
     - wheel.signatures
-    - wheel.test
```
Comment (jakirkham) [inline recipe/meta.yaml:33]: I think this should still be there. 😕

File: recipe/meta.yaml [hunk 3]
```diff
@@ -13,6 +13,7 @@ source:
 
 build:
   number: 0
+  python: noarch
   script: python setup.py install --single-version-externally-managed --record=record.txt
   entry_points:
     - wheel=wheel.tool:main
```
Comment (jakirkham) [inline recipe/meta.yaml]: Oops! Missed this. 😞 Please add spaces around the `=`. Seems `conda-build` sometimes doesn't like this `=` without spaces in `entry_points` of `noarch: python` packages.

**Comment by jakirkham** [comment]
> There are so many commits. If you want I can create a different branch with only two commits (version bump and re-render).

That's ok. Let's just try to finish out this PR.

File: recipe/meta.yaml (hunk 2 above)
Comment (sannykr) [reply recipe/meta.yaml:33]: I had removed that, but added that again. I still think wheel.test should not be there. In older version, there was a wheel.test directory, but not in the latest version.

Comment (jakirkham) [reply recipe/meta.yaml:33]: NVM. Appears I'm wrong. Seemed to be in the source and the `MANIFEST.in`, but not the tarball on PyPI. Sorry for the confusion. Please drop after all.

**Review by jakirkham — APPROVED** [review]

**Comment by jakirkham** [comment]
Closing and reopening [doesn't work]( https://discuss.circleci.com/t/trigger-a-new-build-when-reopening-a-pr/13487 ) with CircleCI unfortunately. Have restarted it manually.

**Comment by sannykr** [comment]
@jakirkham : Could you please review and merge if everything looks okay.

**Comment by jakirkham** [comment]
Thanks @sannykr.

File: recipe/meta.yaml (hunk 1 above)
Comment (jakirkham) [reply recipe/meta.yaml:16]: Fixed in PR ( https://github.com/conda-forge/wheel-feedstock/pull/6 ).

**Comment by jakirkham** [comment]
Finding that the `noarch: python` needs to be reverted to include `wheel` packages in `constructor` installs. So am proposing to revert that change in PR ( https://github.com/conda-forge/wheel-feedstock/pull/9 ).

**Pull request merged**